src/api/app/services/project_status_controller_service/attrib_values_finder.rb

```diff
@@ -0,0 +1,31 @@
+module ProjectStatusControllerService
+  class AttribValuesFinder
+    attr_accessor :packages, :name, :namespace
+
+    def initialize(packages, namespace, name)
+      @packages = packages
+      @name = name
+      @namespace = namespace
+    end
+
+    def self.call
+      raise NotImplementedError
+    end
+
+    def attribute_values
+      AttribValue.where(attrib_id: package_ids_by_attribute_type).joins(:attrib).pluck('attribs.package_id, value')
+    end
+
+    private
+
+    def attribute_type
+      AttribType.find_by_namespace_and_name(@namespace, @name)
```

Comment:
This could be memoized since it might be called twice in `package_ids_by_attribute_type`.

Reply:
please check it again @dmarcoux